docs/dask_cudf/source/api.rst

```diff
@@ -2,74 +2,34 @@
  API reference
 ===============
 
-This page provides a list of all publicly accessible modules, methods,
-and classes in the ``dask_cudf`` namespace.
+As recommended in the introduction, we recommend that you use ``dask_cudf``
+through the `Dask DataFrame`_ API with the backend set to cudf.
 
+.. code-block:: python
+
+   >>> import dask
+   >>> dask.config.set({"dataframe.backend": "cudf"})
+
+With this configuration, ``dask_cudf`` will behave as documented in the Dask
+DataFrame API, but with ``cudf`` objects used in place of ``pandas`` objects.
+
+The rest of this page documents the API you might use from ``dask_cudf``
+explicitly.
 
 Creating and storing DataFrames
 ===============================
 
 :doc:`Like Dask <dask:dataframe-create>`, Dask-cuDF supports creation
-of DataFrames from a variety of storage formats. For on-disk data that
-are not supported directly in Dask-cuDF, we recommend using Dask's
-data reading facilities, followed by calling
-:meth:`*.to_backend("cudf")` to obtain a Dask-cuDF object.
+of DataFrames from a variety of storage formats. In addition to the methods
+documented there, Dask-cuDF provides some cuDF-specific methods:
 
 .. automodule:: dask_cudf
    :members:
       from_cudf,
-      from_delayed,
```

Comment:
Which of these do we want to keep here? For now, I've gone with the rule of "document stuff that's unique to dask-cudf".

I see that I should add `to_orc` and `read_text`, as those don't seem to be in dask's API docs (actually, dask.dataframe has a to_orc, but it's maybe not listed in the API docs. I'll fix that upstream).

Reply:
Yeah - Agree that `read_text` and `from_cudf` are the important APIs to cover.